src/pluralTypes.js

```diff
@@ -23,6 +23,9 @@ const pluralTypes = {
   },
   icelandic(n) {
     return n % 10 !== 1 || n % 100 === 11 ? 1 : 0;
+  },
+  estonian(n) {
+    return n !== 1 ? 1 : 0;
```

Comment:
the estonian plural rule seems to be the same as the german one, can you remove this function and use `et: GERMAN` in pluralMap?

Reply:
Done 🙂